### ISSUE TITLE
Reconsider the ban on NYA companies

### ISSUE THREAD

**Issue by Cobra-Bitcoin** [bitcoin-dot-org/bitcoin.org#2577]
After the whole S2X situation, we removed links to many of the businesses that had signed the NYA. The site doesn't mention Coinbase, Xapo, BitPay, BTC.com, etc. I think enough time has passed for us to reconsider the situation. The removal and subsequent period of not linking to them was reasonable, since these companies could have attempted _another_ contentious hard fork shortly after S2X failed, but time has shown that they've completely failed and given up at taking over Bitcoin, and are unlikely to try again.

I believe that attacks on Bitcoin through manipulating large economic actors like Coinbase are a thing of the past. The next phase of attack is likely to be through miners, and soft forks, and not through contentious hard forks backed by economic weight. I think new users would benefit from at least knowing about some of these services. IMO, we have three reasonable options for how to proceed:

- Add back all the NYA companies we originally removed as normal
- Add back all the NYA companies, but have some sort of hover effect on the link that displays a warning like "This company has promoted a contentious attempt to upgrade Bitcoin in the past"
- Wait a year or two and reassess the situation then
- Never add them back, and continue to promote alternative businesses

Thoughts?

**Comment by crwatkins** [comment]
In regards to the wallet listings, as I have said [before](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1979#issuecomment-354033808)  my major focus is providing a list of wallets that are safe to use. Many of our criteria require wallets to function transparently in ways that minimize the potential harm to users regardless of future decisions of the developers.  I don't have the resources or the aptitude to analyze the potential future decisions of all developers that might list wallets, so I rely on minimizing the impact of unknown future decisions.

Since we have no punitive criteria that I am aware of, I believe that wallets that comply with our current criteria should be listed.

**Comment by Cobra-Bitcoin** [comment]
@crwatkins Agreed, I've re-opened the pull request in question.

I think enough time has passed and the landscape has shifted quite a bit, and that we really need to get out of the hyper-aggressive defensive mindset, in order to better serve users. Obviously if these companies align themselves for further attacks again, we can remove them, but I feel like the risk of that is too small at this point.

**Comment by wbnns** [comment]
> Add back all the NYA companies we originally removed as normal

or

> Wait a year or two and reassess the situation then

Either of these options SGTM. A hover effect warning wouldn't work on mobile devices or tablets (which a significant number of site visitors use) and also might be confusing to a lot of new users.

**Comment by theymos** [comment]
As far as I can tell, if they don't do anything like that again, it'll be because it's become too *difficult* now, not because they've learned from it. So I wouldn't re-add them unless they admit their mistakes and in particular acknowledge that Bitcoin *is not governed*, especially not by miners. Otherwise they very well could do something similar in the right circumstances: perhaps if spearheaded by a massive future ETF, for example.

**Comment by harding** [comment]
I ACK'd the removal of the NYA companies a year ago based on this site's [contentious hard fork policy](https://bitcoin.org/en/posts/hard-fork-policy) which concludes with this text,

> we will not penalize anyone for contributing to a discussion. We will only stop promoting particular wallets and services if they plan to move their users onto a contentious hard fork by default.

AFAIK, no companies are currently planning to do this, so any company that requests to be relisted should be relisted.  @theymos's comments seem to be about penalizing organizations for holding opinions we disagree with, which is something that the policy explicitly said that this site wouldn't do.  So, yeah, I suggest not doing that.

I do kind of like the idea of there being a document that describes in plain English some of the key principles under which Bitcoin works that organizations need to sign in order to be listed on this website, but that seems like a completely different and unrelated policy that should be proposed independently and applied to all listed organizations equally (as all of this site's policies should be applied to all organizations equally).

**Comment by crwatkins** [comment]
I concur with @harding that we should follow our written policies.

**Comment by wbnns** [comment]
Opened #2710.

**Comment by crwatkins** [comment]
Opened #2712.
